### PR TITLE
Handling Redirects from current site to future site via Wagtail

### LINKED ISSUE #450: Redirects
Potentially assigned to Nat? Can we add him to this Github repo https://app.asana.com/0/113884670727555/113884670727555 ?

Need to ensure .net still redirects, while forwarding current .org/program/page structure to new .org/program/content-type/page structure.

### PR DESCRIPTION
**Pull request by sanajaved7**
Using wagtail Redirect model, the script creates objects that map old (current) URLs to new paths

[Resolves #450]

### CODE REVIEW

File: home/management/api/newamerica_api_client.py [hunk 1]
```diff
@@ -165,6 +165,19 @@ def get_in_the_news(self):
 				for post in post_set['results']:
 					yield post, program_id
 
+	def get_posts(self):
+		"""
+		Gets all the content type of Post from the old database API
+		for all programs and creates new objects in the new database 
+		using the Event model
```
Comment (kjacks) [inline home/management/api/newamerica_api_client.py]: @sanajaved7 is it just the event model, or whatever the post type is?

File: home/management/api/redirect.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            for program in post['programs']:
+                old_path = '/{0}/{1}'.format(
+                    mapped_programs[str(program)], post_slug
+                )
+                new_redirect, created = Redirect.objects.get_or_create(
+                    old_path=old_path
+                )
+                new_redirect.redirect_page = new_page
+                new_redirect.save()
+                print(old_path)
```
Comment (kjacks) [inline home/management/api/redirect.py]: @sanajaved7 just flagging this print statement to make sure we want to keep it for debugging, etc.

**Comment by kjacks** [comment]
@sanajaved7 this looks great - are we then handling all redirects through wagtail, or is this just for posts and people?

File: home/management/api/redirect.py (hunk 2 above)
Comment (sanajaved7) [inline home/management/api/redirect.py]: @kjacks yep want to keep it there just to make sure script is working + moving along!

**Comment by sanajaved7** [comment]
@kjacks I think this actually allows us to handle all the redirects through Wagtail actually. I can elaborate more during our check-in but the script creates objects in Wagtail's Redirect model using the old and new path for all pages. Pretty cool right?

**Comment by kjacks** [comment]
@sanajaved7 absolutely! That would be great if we could control it all from here - as long as there are no performance implications? I feel like wagtail would have redirects figured out pretty well though

**Comment by sanajaved7** [comment]
@kjacks Definitely worth continuing to test in staging but for what it's worth, it's a pretty smooth redirect locally at least so I'm hoping that's a good sign 😄

**Pull request merged**